app/scripts/modules/core/authentication/userMenu/userMenu.directive.html

```diff
@@ -0,0 +1,3 @@
+<li ng-show="authEnabled" class="user-menu">
```

Comment:
Totally agree with the `ng-if` actually. The problem I had was when `replace: true` was set on the directive and I ran through the tests the `$compile` portion wouldn't render the dom element and instead returned a html comment `<!-- ng-if authEnabled -->`. I'll look into this more but if you know offhand a solution to this it would be greatly appreciated :)